tests/unit/type/stream-cgroups.tcl

```diff
@@ -9,6 +9,27 @@ start_server {
         set err
     } {BUSYGROUP*}
 
+    test {XGROUP CREATE: with ENTRIESREAD parameter} {
+        r DEL mystream
+	r XADD mystream 1-1 a 1
+	r XADD mystream 1-2 b 2
+	r XADD mystream 1-3 c 3
+	r XADD mystream 1-4 d 4
+        assert_error "*value for ENTRIESREAD must be positive or -1*" {r XGROUP CREATE mystream mygroup $ ENTRIESREAD -3}
+
+	r XGROUP CREATE mystream mygroup1 $ ENTRIESREAD 0
+	r XGROUP CREATE mystream mygroup2 $ ENTRIESREAD 3
+
+	set reply [r xinfo groups mystream]
+        set group_info1 [lindex $reply 0]
+        set entries_read1 [lindex $group_info1 9]
+        assert_equal $entries_read1 0
+
+        set group_info2 [lindex $reply 1]
+        set entries_read2 [lindex $group_info2 9]
+        assert_equal $entries_read2 3
```

Comment:
i think it's better to match the consumer groups properly rather than rely on the index, will be more explicit, but also less fragile? (although they're currently stored in a rax so the order is consistent, right?)

Reply:
Here I follow the same sytax in the test {XGROUP CREATECONSUMER: create consumer if does not exist}
You can see here they are written as below:

![image](https://user-images.githubusercontent.com/51993843/184980365-8d4d891b-3943-408a-942b-03980f6a89a6.png)

I think if it is written explicitly, it is better for later developer to read. I could update it with no harmful.  Thanks

Reply:
the `[lindex $reply 0]` part there is ok, since there's only one group, but here you know one group has 0 entries, and the other group has 3 entries, and you rely on their order in the reply (in the example you gave it doesn't).

in the existing example you gave, the `[lindex $group_info 3]` part is ugly, should have been `[dict get $group_info consumers]`.

same as in your case the explicit reference to field 9 (`lindex $group_info2 9`) is ugly, and i'd rather do `dict get $group_info2 entries-read`. this is:
1. more correct
2. less fragile
3. more readable

i know there are a ton of badly written tests in that respect, and i don't wanna fix them now, but when writing new ones, or modifying existing ones, let's avoid using `lindex` with arbitrary hard coded indexes.

Reply:
I understand and accept your idea. I update the codes in both tcl files, and use "dict get" syntax instead of lindex in some places. Please take a look, Thanks

Reply:
my initial ask was to match the groups by names (not by index `group_info1 [lindex $reply 0]`)
can you look into that too?

Reply:
Sorry Oran, I misunderstand what you said, I thought the statement the [lindex $reply 0] part is ok.
Now I update the test case codes to check the entries-read value according to the group name.
Please take a look, Thanks